src/modules/commandrunner/tests/CommandRunnerTests.cpp

```diff
@@ -320,4 +320,48 @@ namespace Tests
         EXPECT_EQ(MMI_OK, m_commandRunner->Get(m_component, m_reportedObject, &reportedPayload, &payloadSizeBytes));
         EXPECT_TRUE(IsJsonEq(Command::Status::Serialize(status), std::string(reportedPayload, payloadSizeBytes)));
     }
+
+    TEST(CommandRunnerFactory, CreateClients)
```

Comment:
Can we also validate the session count after every client/destroy session?

Reply:
Good idea! Added checks for client count.